### PR TITLE
fix #880 fix cannot open bot Court

### LINKED ISSUE #880: Cannot open event in court
This happens after [no one bets on the event and set result]

image

### PR DESCRIPTION
**Pull request by henryzbw**
image

@BernardHan biubiubiu

### CODE REVIEW

File: src/languageProvider/locales/ko-KR.json [hunk 1]
```diff
@@ -365,6 +366,10 @@
   "withdrawDetail.totalBetTotalVote": "총 베팅 금액 {qtum} QTUM. 총 투표 금액 {bot} BOT.",
   "withdrawDetail.withdrawTo": "철회 대상",
   "withdrawDetail.youBetYouVote": "{qtum} QTUM을 베팅하셨습니다. 당신은 {bot} BOT에 투표했습니다.",
+  "withdrawDialog.amountExceedLimit": "Amount exceed the limit",
```
Comment (BernardHan) [inline src/languageProvider/locales/ko-KR.json]: translate

File: src/languageProvider/locales/zh-Hans.json [hunk 2]
```diff
@@ -365,6 +366,10 @@
   "withdrawDetail.totalBetTotalVote": "共投注 {qtum} QTUM. 共筹集 {bot} BOT.",
   "withdrawDetail.withdrawTo": "提现到",
   "withdrawDetail.youBetYouVote": "您投入了 {qtum} QTUM. 您投入了 {bot} BOT.",
+  "withdrawDialog.amountExceedLimit": "Amount exceed the limit",
+  "withdrawDialog.amountLargerThanZero": "Amount should be larger than 0",
```
Comment (BernardHan) [inline src/languageProvider/locales/zh-Hans.json]: translate

File: src/scenes/Event/components/ResultHistory/index.js [hunk 3]
```diff
@@ -74,7 +74,7 @@ export default class EventResultHistory extends Component {
               </TableRow>
             </TableHead>
             <TableBody>
-              {_.map(sortedOracles, (oracle, index) => {
+              {_.map(filteredOracles, (oracle, index) => {
```
Comment (BernardHan) [inline src/scenes/Event/components/ResultHistory/index.js]: good job :)

File: src/languageProvider/locales/ko-KR.json (hunk 1 above)
Comment (henryzbw) [reply src/languageProvider/locales/ko-KR.json]: beat you! @ippudo will do.

**Pull request merged**